std/php7/Boot.hx

```diff
@@ -61,6 +61,13 @@ class Boot {
 	}
 
 	/**
+		Builds Haxe Array from array declaration syntax.
+	**/
+	public static function array( arr:NativeIndexedArray<Dynamic> ) : Array<Dynamic> {
```

Comment:
That was my original idea.
BTW: Don't you need to rename `Array` to `HxArray` or something?